lib/AppInfo/Application.php

```diff
@@ -147,6 +147,7 @@ protected function registerCollaborationResourceProvider(IServerContainer $serve
 		$resourceManager->registerResourceProvider(ConversationProvider::class);
 		$server->getEventDispatcher()->addListener('\OCP\Collaboration\Resources::loadAdditionalScripts', function () {
 			\OCP\Util::addScript(self::APP_ID, 'collections');
+			\OCP\Util::addScript(self::APP_ID, 'deck');
```

Comment:
@juliushaertl is this the expected way to handle it?

Reply:
Is a bit to wide i'd say as we don't need this integration in files for example. For the complementary implementation of message actions i picked loading this in a BeforeTemplateRendered event: https://github.com/nextcloud/deck/pull/2795/files#diff-848ad1b5ff7f00ceb7ca05dc2abe8b592bb6711f3df7f01ae2b567508fe6906c

That is of course still somehow relying on the talk app route but felt cleaner then adding a dedicated event in the OCA namespace. But I have no strong oppinions on that.

Reply:
adjusted using a similar technique. See DeckLoaderPlugin.php